### PR TITLE
Depend directly on remill

### PR DESCRIPTION
**Pull request by pgoodman**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -47,8 +47,8 @@ WORKDIR /anvill
 COPY . ./
 
 ENV PATH="${LIBRARIES}/llvm/bin:${LIBRARIES}/cmake/bin:${LIBRARIES}/protobuf/bin:${PATH}"
-ENV CC="${LIBRARIES}/llvm/bin/clang"
-ENV CXX="${LIBRARIES}/llvm/bin/clang++"
```
Comment (ekilmer) [inline Dockerfile:51]: Why aren't we using the version of `clang` from `cxx-common`?

Comment (pgoodman) [reply Dockerfile:51]: cxx-common is built with `/usr/bin/clang`, so I think it's best to have everything consistent.

Comment (ekilmer) [reply Dockerfile:51]: Okay, that should be doable. I'll try to get this working in Docker

Comment (ekilmer) [reply Dockerfile:51]: I pushed a commit to install `clang` during the `deps` stage of Docker, but it seems there are some other errors now regarding dependencies.

**Pull request merged**